### ISSUE TITLE
Prison lab escapes are likely unwinnable

### ISSUE THREAD

**Issue by Xhuis** [CleverRaven/Cataclysm-DDA#25538]
**Describe the bug**  
As of the (otherwise excellent) #24410 PR, prisons now have a chance to have hidden laboratories beneath them. While this is interesting and makes for more variety, I've run into a problem: the turret hallway leading into the lab is very short, and will end with the player being instantly shot upon ascending the steps.

**To Reproduce**  
1. Find a prison lab, or artificially make one with overmap editor through the `hidden_stairs` search.
2. Stand on the stairs, or ascend them as a lab escapee.
3. Get shot and die instantly.

**Expected behavior**  
The turret hallway should be long enough to allow the player to actually move around, let alone not get shot instantly. This makes labs inescapable.

**Screenshots**  
## Normal lab stairs:
![image](https://user-images.githubusercontent.com/15898043/45350653-4f8fcc00-b582-11e8-853b-99e4b300ead5.png)
## Prison lab stairs:
![image](https://user-images.githubusercontent.com/15898043/45350663-561e4380-b582-11e8-9f11-a711cccd1eb6.png)

On the run I recently had cut short by this error, I think the hallway was one or two tiles shorter, as I was shot in the head for 50 damage immediately upon ascending the steps. After sleeping off the damage, I made a matchbook bomb, lit it downstairs, then ascended and quickly attempted to hurl it at the turret, to no avail; I was immediately shot to death without moving away from the stairs.

**Versions and configuration(please complete the following information):**
 - OS: [e.g. iOS 8 or Windows 10 or Ubuntu 18.04]: Windows 10
 - Game Version [from the main menu, e.g. 0.C-29938-g90f5268437]: 0.C-31903-gf9f6e81
 - Graphics version [Tiles or Terminal]: Tiles
 - Mods loaded [e.g.dda, boats, hacktheplanet, StatsThroughSkills]: None affecting this

**Additional context**  
Unfortunately, it didn't occur to me to take a screenshot when I lost my actual run, as I was frustrated.

**Comment by mrkybe** [comment]
Go back down, dig your way out?  Not sure if you can craft the necessary digging tools though?

**Comment by Xhuis** [comment]
A pickaxe requires a forge and smithing tools, which I don't think is reasonably obtainable, especially given the staggering fabrication cost.

A jackhammer requires an engine and gasoline, which can't be obtained in a lab.

An electric jackhammer requires welding supplies and a very high electronics skill, plus the recipe is found in high-level mechanics and electronics books. More obtainable than the other two, but not guaranteed.

On top of that, mining an upstair requires a helmet, 6 fabrication, 4 survival, and 8 hours, plus a long rope and 8 hours. Compared to other lab escapes, the amount of effort required is staggeringly disproportional based on luck alone.

**Comment by AbsalomAchitophel526** [comment]
I was able to get past the turret but my 'solution' was convoluted to say the least.  I had to craft whatever armor I could for my torso and head and layer it all, despite encumbrance, leaving arms and legs free.  I then primed an EMP grenade at the bottom of the stairs, waited until it was two turns before detonation, then I went up and threw the grenade.

Unfortunately, it was still two turns when I reached the top.  I threw it so that the EMP grenade would not reach the card reader, got shot, then went downstairs.  I then slept and healed torso and head damage only, then ascended the stairs again.  The grenade ticked down another turn, I got shot in the head, barely surviving, then went downstairs to heal again.  It took me two weeks of healing and popping up the stairs before the grenade detonated, killing the turret.

Of course my electrohack then broke the card reader so it was all for naught, but still, it's do-able.
